piper/resolutionspage.py

```diff
@@ -12,6 +12,9 @@
 from gi.repository import Gtk  # noqa
 
 
+UNSUPPORTED_REPORT_RATE_SENTINEL = 0
```

Comment:
This is wrong.

Per libratbag API docs: 

ReportRate
uint for the report rate in Hz assigned to this profile. This rate must be one of those listed in [ReportRates](https://libratbag.github.io/dbus.html#ReportRates).

ReportRates
This list may be empty if the device does not support reading and/or writing to resolutions.

Reply:
Probably just an oversight in the docs. Not only that, ReportRates may never be empty because the device will fail sanity check otherwise.